examples/scala/build.earth

```diff
@@ -0,0 +1,20 @@
+FROM hseeberger/scala-sbt:11.0.7_1.3.13_2.11.12
+WORKDIR /scala-example
+
+deps:
+    COPY build.sbt ./
+    COPY project project
+    RUN sbt update
+    SAVE IMAGE
+
+build:
+    FROM +deps
+    COPY src src
+    RUN sbt assembly
+    SAVE ARTIFACT target/scala-2.13 /scala-example AS LOCAL build/bin
+    SAVE IMAGE
+
+docker:
+    COPY build/bin build/bin
```

Comment:
Here you probably want to copy from the `+build` target rather than from the local dir, so that you can build `+docker` in one go.

```suggestion
    COPY +build/scala-example build/bin
```

Reply:
Ah, makes sense.  This line was actually the trickiest part for me.  Fixed.